### ISSUE TITLE
Make the cookie text white 

### ISSUE THREAD

**Issue by Nishim12** [ubyssey/ubyssey-dispatch-theme#1516]
Make the cookie text white from black as it improves the color contrast and makes the color contrast to AAA which helps in accessibility.
